### ISSUE TITLE
File input/output

### ISSUE THREAD

**Issue by ehmicky** [sindresorhus/nano-spawn#19]
Passing a file to `stdin`/`stdout`/`stderr` can be done using `fs.createReadStream()` and `fs.createWriteStream()`.

Implementing it is not 100% trivial as it requires also: 
- Potentially, allowing both file paths and file URLs
- Listening for stream errors
- Cleaning up the stream if the subprocess fails

So I would suggest not implementing it. Thoughts?

**Comment by sindresorhus** [comment]
Yeah, we can leave it out for now. But I prefer to keep this issue open.

**Comment by sindresorhus** [comment]
Maybe worth opening an issue on Node.js about this? Would be useful if `stdio[n]` accepted:

- `Uint8Array`: For when you already have a loaded buffer.
- `FileHandle`: ?
- `WebStream`: For completeness.
